packages/enzyme-utilities/src/test/index.test.tsx

```diff
@@ -34,4 +36,31 @@ describe('enzyme-utilities', () => {
       expect(toggle.find('button').text()).toBe('inactive');
     });
   });
+
+  describe('findById', () => {
+    it('finds an element with a matching ID', () => {
+      const matchedNode = <div id="foo" />;
+      const found = findById(
+        mount(
+          <div>
+            {matchedNode}
+            <div id="bar" />
+          </div>,
+        ),
+        'foo',
+      );
+
+      expect(found.getElement()).toEqual(matchedNode);
+    });
+
+    it('only returns an outer component when it renders its own children with the same ID', () => {
```

Comment:
I think this wants to be two tests, this one and one that is expected not to find.

Reply:
I don't think I understand, not sure what the second test would look like? This is just meant to make sure we don't get the rendered `input`s when this utility finds a `TextField`/ `Checkbox`. I can do a test for no matches but IMO it's not a super meaningful test